### PR TITLE
Software agency

### PR DESCRIPTION
**Pull request by Malekalarmash**
-Remove unwanted spaces
-Move the styling to the software-agency.css
-Fix typo

### CODE REVIEW

File: public/pages/intro-class.html [hunk 1]
```diff
@@ -211,13 +212,13 @@ <h4>Refcode Bootcamp</h4>
           <p class="timeframe">(<i>3 months</i>)</p>
           <p>Graduates are eligible to apply for the Refcode Bootcamp, where you will learn additional coding skills. You can also apply to be a Refcode Bootcamp Resident. Residents are paid a stipend of $1,500 per month.</p>
 
-          <h4>Refcode Software Agency Technician</h4>
+          <h4>Refcode Software Agency  Technician</h4>
           <p class="timeframe">(<i>6 months - 1 year</i>)</p>
```
Comment (brentonstrine) [inline public/pages/intro-class.html:216]: space

File: public/pages/intro-class.html [hunk 2]
```diff
@@ -211,13 +212,13 @@ <h4>Refcode Bootcamp</h4>
...
           <p>After the bootcamp, participants can apply for a job as a Software Technician at the Refcode Agency. In this program, you will be paid to build sophisticated applications and to grow and develop your skills. Participants who do well will be promoted quickly to Technician II, and Technician III.  Software Technicians are paid $11 - 13 per hour depending on level.</p>
 
-          <h4>Refcode Software Agency Engineer</h4>
+          <h4>Refcode Software Agency  Engineer</h4>
```
Comment (brentonstrine) [inline public/pages/intro-class.html:219]: space

File: public/pages/intro-class.html [hunk 3]
```diff
@@ -211,13 +212,13 @@ <h4>Refcode Bootcamp</h4>
...
           <p class="timeframe">(<i>6 months - 1 year</i>)</p>
-          <p>Those who have been promoted to Software Technician III will be eligible to apply for a Software Engineer I role at the Refcode Software Agency. Engineers take on greater ownership of more complex tasks and will need to show higher creativity and independent problem solving skills. Engineers who do well will be quickly promoted to Software Engineer II and Software Engineer III. Software Engineers are paid $14 - 18 per hour, depending on level and experience.</p>
+          <p>Those who have been promoted to Software Technician III will be eligible to apply for a Software Engineer I role at the Refcode Software Agency . Engineers take on greater ownership of more complex tasks and will need to show higher creativity and independent problem solving skills. Engineers who do well will be quickly promoted to Software Engineer II and Software Engineer III. Software Engineers are paid $14 - 18 per hour, depending on level and experience.</p>
```
Comment (brentonstrine) [inline public/pages/intro-class.html:221]: there's a space here `Agency .`

**Review by brentonstrine — APPROVED** [review]
no blockers

File: public/pages/software-agency.html [hunk 4]
```diff
@@ -54,11 +54,12 @@
...
-      <div class="banner-text software-agency">Refcode Software Agency </div>
+      <div class="banner-text software-agency">Refcode Software Agency</div>
     </div>
-    <h2 style="margin-top: 150px;">Refcode Software Agency </h2>
-        <p>Refcode Software Agency  is a work space where Refcode graduates and students will come together to experience a real, professional projects. With the help of Softwre Engineers, we will provide
-        you witha a great, outstanding product. Contact us to learn more!</p>
+    <h2 style="margin-top: 150px;">Refcode Software Agency</h2>
+        <p>Refcode Software Agency is a safe learning environment where Refcode graduates and advisors come together to experience a real, professional projects. 
+          When you hire Refcode Software Agency, you don't only get your website deployed, you will also be part of the journey of refugees and immigrants who want to become future Software Engineers. 
+          Contact us to learn more!.</p>
```
Comment (brentonstrine) [inline public/pages/software-agency.html:62]: !.

Comment (Malekalarmash) [reply public/pages/software-agency.html:62]: What do you mean?

**Review by brentonstrine — APPROVED** [review]
Merge

**Pull request merged**